src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -65,7 +65,7 @@ def load_arguments(self, _):
         c.argument('location', arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
         c.argument('image', validator=validate_image, help='The container image name')
         c.argument('cpu', type=float, help='The required number of CPU cores of the containers, accurate to one decimal place')
-        c.argument('memory', type=float, help='The required memory of the containers in GB, accurate to one decimal place')
+        c.argument('memory', type=float, help='The required memory of the containers in GiB, accurate to one decimal place')
```

Comment:
This might be more accurate, but the Azure's doc has used the `GB` across, like [here](https://docs.microsoft.com/en-us/rest/api/container-instances/containergroups/createorupdate#container) and [there](https://docs.microsoft.com/en-us/rest/api/compute/disks/createorupdate). Unless there are users get confused, we should hold on this till doc gets updated first

Reply:
Also, if you buy a physical machine, it is also the [GB](https://www.microsoft.com/en-us/store/config/Surface-Book-2/8MCPZJJCC98C?crosssellid=surfacebook2interstitial&selectedColor=&preview=&previewModes=) which gets mentioned

Reply:
@yugangw-msft 

> Also, if you buy a physical machine, it is also the GB which gets mentioned

When you by a physical disk, it's sized in GB, but when you allocate an Azure Disk, it's sized in GiB. Showing GB in these places is extremely misleading, because it does not accurately reflect the underlying size.